src/sharedHooks/useAuthenticatedInfiniteQuery.ts

```diff
@@ -1,29 +1,51 @@
 import { useRoute } from "@react-navigation/native";
-import { useInfiniteQuery } from "@tanstack/react-query";
+import { QueryKey, useInfiniteQuery, UseInfiniteQueryOptions } from "@tanstack/react-query";
 import { getJWT } from "components/LoginSignUp/AuthenticationService";
 import i18n from "i18next";
 import { handleRetryDelay, reactQueryRetry } from "sharedHelpers/logging";
 import { useCurrentUser } from "sharedHooks";
 
+interface QueryFunctionOptions {
+  api_token: string | null;
+  locale?: string;
+}
+
+type QueryFunction<T> = ( params: {
+  pageParam: number,
+  locale?: string
+}, options: QueryFunctionOptions ) => Promise<T>;
+
+type QueryOptions<TQueryFnData, TData> = Omit<UseInfiniteQueryOptions<
+  TQueryFnData,
+  Error,
+  TData,
+  TQueryFnData,
+  QueryKey,
+  number
+>, "queryKey"> & {
+  allowAnonymousJWT?: boolean;
+}
+
 // Should work like React Query's useInfiniteQuery with our custom reactQueryRetry
 // and authentication
-const useAuthenticatedInfiniteQuery = (
-  queryKey: Array<string>,
-  queryFunction: Function,
-  queryOptions: object = {}
```

Comment:
Functional change 1: Removal of ` = {}` because `getNextPageParam` is a required option parameter, so `{}` is invalid. I confirmed all callers pass `queryOptions`.